src/test/java/duke/parser/ParserTest.java

```diff
@@ -0,0 +1,264 @@
+package duke.parser;
+
+import duke.command.AddTodoCommand;
+import duke.command.AddDeadlineCommand;
+import duke.command.AddEventCommand;
+import duke.command.MarkCommand;
+import duke.command.UnmarkCommand;
+import duke.command.DeleteCommand;
+import duke.command.ListCommand;
+import duke.command.ExitCommand;
+import duke.command.WrongCommand;
+import duke.exception.DukeException;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class ParserTest {
+
+	@Test
+	public void success_AddTodo() {
```

Comment:
Same problem as mentioned before. The tab size seems a bit off. Maybe double check to ensure your tab size is consistent throughout the project files?